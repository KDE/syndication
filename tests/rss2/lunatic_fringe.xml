<?xml version="1.0" encoding="utf-8"?>
<!-- heavy use of entities, hebrew letters
-->
<!-- generator="wordpress/2.0.2" -->
<rss version="2.0" 
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	>

<channel>
	<title>The Lunatic Fringe</title>
	<link>http://tim.geekheim.de</link>
	<description>Tim Pritlove</description>
	<pubDate>Thu, 02 Nov 2006 01:31:22 +0000</pubDate>
	<generator>http://wordpress.org/?v=2.0.2</generator>
	<language>en</language>
<item>
		<title>Kann hier jemand Hebr&#228;isch?</title>
		<link>http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/</link>
		<comments>http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/#comments</comments>
		<pubDate>Wed, 01 Nov 2006 19:46:56 +0000</pubDate>
		<dc:creator>Tim Pritlove</dc:creator>
		
	<category>German</category>
		<guid isPermaLink="false">http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/</guid>
		<description><![CDATA[
Jemand des Hebr&#228;ischen m&#228;chtig? W&#252;rde ja gerne mal wissen, was Haaretz so &#252;ber den CCCB zu berichten wei&#223;: Haaretz: &#1508;&#1497;&#1512;&#1488;&#1496;&#1497;&#1501; &#1506;&#1501; &#1502;&#1497;&#1509; &#1514;&#1508;&#1493;&#1495;&#1497;&#1501;

]]></description>
			<content:encoded><![CDATA[<p>
Jemand des Hebr&auml;ischen m&auml;chtig? W&uuml;rde ja gerne mal wissen, was Haaretz so &uuml;ber den CCCB zu berichten wei&szlig;: <a href="http://www.haaretz.co.il/captain/pages/ShArtCaptain.jhtml?contrassID=11&amp;subContrassID=4&amp;itemNo=772754">Haaretz: &#1508;&#1497;&#1512;&#1488;&#1496;&#1497;&#1501; &#1506;&#1501; &#1502;&#1497;&#1509; &#1514;&#1508;&#1493;&#1495;&#1497;&#1501;</a>
</p>
]]></content:encoded>
			<wfw:commentRSS>http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/feed/</wfw:commentRSS>
		</item>
	</channel>
</rss>
