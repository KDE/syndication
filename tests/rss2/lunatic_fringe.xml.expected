# Feed begin ######################
title: #The Lunatic Fringe#
link: #http://tim.geekheim.de#
description: #Tim Pritlove#
language: #en#
# Item begin ######################
id: #http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/#
title: #Kann hier jemand Hebr√§isch?#
link: #http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/#
description: #Jemand des Hebr&#228;ischen m&#228;chtig? W&#252;rde ja gerne mal wissen, was Haaretz so &#252;ber den CCCB zu berichten wei&#223;: Haaretz: &#1508;&#1497;&#1512;&#1488;&#1496;&#1497;&#1501; &#1506;&#1501; &#1502;&#1497;&#1509; &#1514;&#1508;&#1493;&#1495;&#1497;&#1501;#
content: #<p>
Jemand des Hebr&auml;ischen m&auml;chtig? W&uuml;rde ja gerne mal wissen, was Haaretz so &uuml;ber den CCCB zu berichten wei&szlig;: <a href="http://www.haaretz.co.il/captain/pages/ShArtCaptain.jhtml?contrassID=11&amp;subContrassID=4&amp;itemNo=772754">Haaretz: &#1508;&#1497;&#1512;&#1488;&#1496;&#1497;&#1501; &#1506;&#1501; &#1502;&#1497;&#1509; &#1514;&#1508;&#1493;&#1495;&#1497;&#1501;</a>
</p>#
datePublished: #Wed Nov 1 20:46:56 2006#
dateUpdated: #Wed Nov 1 20:46:56 2006#
# Person begin ####################
name: #Tim Pritlove#
# Person end ######################
# Category begin ##################
term: #German#
# Category end ####################
commentsLink: #http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/#comments#
commentsFeed: #http://tim.geekheim.de/2006/11/01/kann-hier-jemand-hebrisch/feed/#
# Item end ########################
# Feed end ########################

